pkgs/tools/networking/unbound/default.nix

```
@@ -107,6 +108,15 @@ stdenv.mkDerivation rec {
 
   doCheck = true;
 
+  patches = [
+    # Fix CVE-2022-30698 and CVE-2022-30699
+    (fetchpatch {
+      url = "https://github.com/NLnetLabs/unbound/commit/f6753a0f1018133df552347a199e0362fc1dac68.patch";
```

```suggestion
      name = "CVE-2022-30698.CVE-2022-30699.patch";
      url = "https://github.com/NLnetLabs/unbound/commit/f6753a0f1018133df552347a199e0362fc1dac68.patch";
```

Give vulnix a chance at detecting the fix.